### ISSUE TITLE
Doesn't digitize

### ISSUE THREAD

**Issue by TorokLev** [Tereshchenkolab/paper-ecg#14]
For some reason, tool doesn't digitize the selected windows of leads from a jpg.
It doesn't say what was wrong. It doesn't have a log. It remaints silent when pushing the "Process" button.
This is very disappointing.

**Comment by tereshchenko9** [comment]
It means poor quality of image. Poor-quality images cannot be digitized. You can try to check leads one by one, play with the bounding box,  and try to select different onset and different size/hight of a bounding box. Sometimes it helps to have a bounding box that is very close to the ECG waveform.

**Comment by TorokLev** [comment]
No. 
I didnt follow your instruction on the installation as it requires my own system environment to modify which I am not going to do but I was happy to be able to start you tool in another way.
I also see that the installation of pyenv crashed.
Also, the I changed the scipy version in the requirements because it wanted to compile a numpy which didn't work out either.
But I managed to start it in virtualenv but the UX doesnt show the possible algorithms. So it hasn't found any.
The interface should let the user know about it.
Yet I don't know how to persuade the UX to find the algorithms. Any idea?
